### PR TITLE
Add DevX Digest 1 onto the website

### PR DESCRIPTION
**Pull request by pawlean**
We released DevX Digest yesterday 🎉 and wanted to get this content on the blog too.   

This PR also adds me to the list of authors. :) 

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/819"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/819"></a>

### CODE REVIEW

**Review by ChristinFrohne — CHANGES_REQUESTED** [review]
Looks awesome! Thanks for sticking to the [design guidelines](https://www.notion.so/Blogging-34c2e078569b4174b42784854f42549f) 🎯 🎉

**Minor comments:**

- [ ] it's best to open external links in a new tab (e.g. Twitter handles or YouTube links). EDIT: I just noticed that's a general issue for all blog posts. Could you fix that @nisarhassan12 ? 🙏

- [ ] you still need a preview text 
Bildschirmfoto 2021-08-05 um 14 14 55

**Comment by pawlean** [comment]
> Looks awesome! Thanks for sticking to the [design guidelines](https://www.notion.so/Blogging-34c2e078569b4174b42784854f42549f) 🎯 🎉
> 
> **Minor comments:**
> 
> * [ ]  it's best to open external links in a new tab (e.g. Twitter handles or YouTube links). EDIT: I just noticed that's a general issue for all blog posts. Could you fix that @nisarhassan12 ? 🙏
> * [x]  you still need a preview text
> 
> Bildschirmfoto 2021-08-05 um 14 14 55

Thanks Christin, always good to get another pair of eyes. 👀  I've updated the excerpt now!

**Review by mikenikles — APPROVED** [review]
Excellent, I'll solicit your feedback about the code structure very soon :).

One nitpick, could you use a lower case file name for `static/images/blog/the-evolution-of-devx/Source.png` please?!

Otherwise, LGTM 👍

**Comment by nisarhassan12** [comment]
> - [ ] it's best to open external links in a new tab (e.g. Twitter handles or YouTube links). EDIT: I just noticed that's a general issue for all blog posts. Could you fix that @nisarhassan12 ? 🙏

Thanks, @ChristinFrohne for bringing this up. For the author avatars and social share buttons, they already open up a new tab I addressed those in some recent PRs. For the rest of the links within the post body I'll make a PR soon.

**Comment by nisarhassan12** [comment]
I'll take over this Pr to update the layout as asked in https://github.com/gitpod-io/website/issues/827

**Comment by pawlean** [comment]
> I'll take over this Pr to update the layout as asked in #827

Cheers! Let me know if you need anything from me.

File: src/components/blog/digest-header.svelte [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+<style type="text/postcss">
+  header {
+    @apply mb-0 !important;
+  }
+</style>
+
+<header class="mb-0">
+  <h1>DevX Digest</h1>
+  <p>Short explanation</p>
```
Comment (ChristinFrohne) [inline src/components/blog/digest-header.svelte]: please use the same font size as for the description on /changelog. -> 16px instead of 18px

Comment (ChristinFrohne) [inline src/components/blog/digest-header.svelte]: @pawlean here you should add a short explanation about the DevX Digest

File: src/components/twitter-follow-button.svelte [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+  let clazz = "";
+  export { clazz as class };
+  export let trackingContext: string;
+</script>
+
+<a
+  href="https://www.twitter.com/gitpod"
```
Comment (ChristinFrohne) [inline src/components/twitter-follow-button.svelte:8]: @nisarhassan12 the twitter link should open in a new tab. 
@pawlean we could also consider using the DevX Conf handle. Let's discuss that and if needed adjust that in a follow-up PR

**Review by ChristinFrohne — COMMENTED** [review]
Just a few inlined comments. Feel free to merge the PR after they have been addressed. Thanks!

File: src/components/blog/digest-header.svelte (hunk 1 above)
Comment (nisarhassan12) [reply src/components/blog/digest-header.svelte]: She already did that here https://github.com/gitpod-io/website/pull/819/commits/d0f5d4b557c258036d458ca413d6fdf572e60d1f Maybe the deploy preview was not ready when you were reviewing.

**Pull request merged**